spec/post_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Koko') }
+
+  before { subject.save }
+
+  describe 'validation tests' do
+    it 'name should be present' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+    end
```

Comment:
## REQUIRED

Kindly include tests for all the validations you added.

-  In the Post spec there should be a test for the `Title must not exceed 250 characters` validation.
- Any custom methods in the models should have a corresponding unit test. Please add corresponding tests for the methods in the `User` and `Post` specs.

![image](https://github.com/Peter-Dumbari/Blog-Application/assets/38088818/dc194f95-211d-4bbc-9d90-90d69f102e14)